sdk/src/main/java/io/dapr/actors/AbstractDaprClient.java

```diff
@@ -111,30 +97,21 @@ private final void tryInvokeAPI(String method, String urlString, String json, fi
         .addHeader(Constants.HEADER_DAPR_REQUEST_ID, requestId)
         .build();
 
-    this.httpClient.newCall(request).enqueue(new Callback() {
 
-      @Override
-      public void onFailure(Call call, IOException e) {
-        cb.onFailure(call, e);
-      }
-
-      @Override
-      public void onResponse(Call call, Response response) throws IOException {
-        try (ResponseBody responseBody = response.body()) {
-          if (!response.isSuccessful()) {
-            DaprError error = parseDaprError(response.body().string());
-            response.close();
-            if ((error != null) && (error.getErrorCode() != null) && (error.getMessage() != null)) {
-              throw new DaprException(error);
-            }
-          } else {
-            String respBodyString = responseBody.string();
-            cb.onSuccess(respBodyString);
-            response.close();
+    try (Response response = this.httpClient.newCall(request).execute()) {
```

Comment:
Revert this change to the previous solution. This one is a blocking operation while the previous one was async.